lib/vantiv_lite/version.rb

```diff
@@ -2,8 +2,8 @@
 
 module VantivLite
   MAJOR = 0
-  MINOR = 2
-  TINY  = 9
+  MINOR = 3
+  TINY  = 2
```

Comment:
This is probably fine, but I'm curious why did you have to change this?

Reply:
We're still trying to figure out how we want to do the versions, but the idea was originally because this was for Ruby 3.2 to have it kind of match. 

I'm ok with changing this to something else if you want because this pr isn't upgrading to ruby 3 anymore. But because it's kind of a bigger change we should still increment the version so we can fallback to the old version of the gem if it doesn't work properly.